interface/src/app/node/node.tpl.html

```diff
@@ -7,13 +7,21 @@
   <div class="container">
     <div class="row">
       <div class="col-xs-6">
-        <high-weight-genes node-id="{{ctrl.id}}"></high-weight-genes>
+        <high-weight-genes node-id="{{ctrl.id}}" h-w-genes="ctrl.hWGenes"></high-weight-genes>
```

Comment:
Since the directive is called "high-weight-genes* already, does it make sense to simply write:
```
<high-weight-genes node-id="{{ctrl.id}}" genes="ctrl.hWGenes">
```
Inside the controller, to make the variable name more descriptive, we can change it from "hWGenes" to "highWeightGenes", or something like this.